src/ideascale/models/custom_fields.rs

```diff
@@ -24,7 +24,10 @@ impl Default for CustomFieldTags {
             proposal_goal: "how_does_success_look_like_".to_string(),
             proposal_metrics: "key_metrics_to_measure".to_string(),
             proposal_public_key: "ada_payment_address".to_string(),
-            proposal_funds: "requested_funds".to_string(),
+            proposal_funds: vec![
+                "requested_funds".to_string(),
```

Comment:
Is this needed by ideascale? Otherwise I imagine this to be a mutually exclusive choice (but it's probably harder to deserialize)

Reply:
this is a solution for coti challenge, which has different field for proposal_funds.